### PR TITLE
Explain plan

### PR DESCRIPTION
**Pull request by ilovesoup**
Add explain plan support

---
This change is [Reviewable](https://reviewable.io/reviews/pingcap/tikv-client-lib-java/115)

### CODE REVIEW

**Review by Novemser — COMMENTED** [review]
Nice

File: src/main/java/com/pingcap/tikv/codec/TableCodec.java [hunk 1]
```diff
@@ -74,6 +74,15 @@ public static ByteString encodeRowKeyWithHandle(long tableId, long handle) {
     return cdo.toByteString();
   }
 
+  public static long decodeRowKey(ByteString rowKey) {
+    CodecDataInput cdi = new CodecDataInput(rowKey);
```
Comment (Novemser) [inline src/main/java/com/pingcap/tikv/codec/TableCodec.java]: If rowKey is null, ```CodecDataInput cdi = new CodecDataInput(rowKey);``` will throw NPE

File: src/main/java/com/pingcap/tikv/codec/TableCodec.java [hunk 2]
```diff
@@ -74,6 +74,15 @@ public static ByteString encodeRowKeyWithHandle(long tableId, long handle) {
...
+    cdi.skipBytes(TBL_PREFIX.length);
+    IntegerType.readLong(cdi);
+    cdi.skipBytes(REC_PREFIX_SEP.length);
```
Comment (Novemser) [inline src/main/java/com/pingcap/tikv/codec/TableCodec.java]: Notice if we need to skip ```IDX_PREFIX_SEP``` further

**Review by Novemser — COMMENTED** [review]
Minor modification

File: src/main/java/com/pingcap/tikv/codec/TableCodec.java (hunk 1 above)
Comment (ilovesoup) [reply src/main/java/com/pingcap/tikv/codec/TableCodec.java]: fixed.

**Review by Novemser — APPROVED** [review]
LGTM

**Pull request merged**